Uplift/Controllers/HomeViewController+Extensions.swift

```diff
@@ -11,13 +11,15 @@ import UIKit
 extension HomeViewController: UICollectionViewDataSource {
 
     func numberOfSections(in collectionView: UICollectionView) -> Int {
+        print("these are the sections \(sections.count)")
```

Comment:
Let's delete this print statement since I don't think it adds any value but is more used for debugging